### PR TITLE
Update Code Model to support ReadOnly and WriteOnly VB auto-properties

### PR DESCRIPTION
**Pull request by DustinCampbell**
VB 14 supports ReadOnly and WriteOnly auto-properties. Code Model's
CodeProperty2.ReadWrite should return the correct value for them. Fixes issue #851.

### CODE REVIEW

File: src/VisualStudio/Core/Test/CodeModel/VisualBasic/CodePropertyTests.vb [hunk 1]
```diff
@@ -737,23 +737,47 @@ End Class
             TestReadWrite(code, EnvDTE80.vsCMPropertyKind.vsCMPropertyKindWriteOnly)
         End Sub
 
-        <ConditionalFact(GetType(x86)), Trait(Traits.Feature, Traits.Features.CodeModel)>
-        Public Sub ReadWrite4()
-            Dim code =
+		<ConditionalFact(GetType(x86)), Trait(Traits.Feature, Traits.Features.CodeModel)>
```
Comment (pharring) [inline src/VisualStudio/Core/Test/CodeModel/VisualBasic/CodePropertyTests.vb]: Are the tabs/spaces correct here?

**Comment by pharring** [comment]
Check your tabs/spaces setting, but otherwise LGTM.

**Comment by Pilchie** [comment]
Ditto @pharring's comment about tabs/spaces, but once that's fixed, :shipit:

**Comment by DustinCampbell** [comment]
Yeah, I'm not sure what happened with the whitespace. Will fix before merging.

**Comment by DustinCampbell** [comment]
OK. Tabs/spaces fixed. Looks like my IDE got switched over to tabs somehow and I didn't have visible whitespace on.

**Pull request merged**